external-import/recordedfuture-notes/src/rflib/rf_notes_to_stix2.py

```diff
@@ -420,21 +505,131 @@ def from_json(self, note):
+                if type_ in [
+                    "IpAddress",
+                    "InternetDomainName",
+                    # "URL", // bug on the RF API when fetching the score of a url
+                    "Hash",
+                ]:
+                    risk_score = None
+                    if self.risk_threshold:
+                        # If a min threshold was defined, we ignore the indicator if the score is lower than the defined threshold
+                        risk_score = self.rfapi.get_risk_score(
+                            INDICATOR_TYPE_URL_MAPPER[type_], name
+                        )
+                        if risk_score < self.risk_threshold:
+                            self.helper.log_info(
+                                f"Ignoring entity {name} as its risk score is lower than the defined risk threshold"
+                            )
+                            continue
+                    if self.risk_as_score:
+                        # We get the risk_score if it was already set. Otherwise, we get it from the API
+                        rf_object.risk_score = (
+                            risk_score
+                            if risk_score
+                            else self.rfapi.get_risk_score(
+                                INDICATOR_TYPE_URL_MAPPER[type_], name
+                            )
+                        )
+                stix_objs = rf_object.to_stix_objects()
             self.objects.extend(stix_objs)
         if "attachment_content" in attr:
             rule = DetectionRule(
-                attr["attachment"], attr["attachment_type"], attr["attachment_content"]
+                attr["attachment"],
+                attr["attachment_type"],
+                attr["attachment_content"],
+                self.author,
             )
             self.objects.extend(rule.to_stix_objects())
 
+    RELATIONSHIPS_MAPPER = [
+        {
+            "from": "threat-actor",
+            "to": [
+                {"entity": "malware", "relation": "uses"},
+                {"entity": "vulnerability", "relation": "targets"},
+                {"entity": "attack-pattern", "relation": "uses"},
+                {"entity": "location", "relation": "targets"},
+                {"entity": "identity", "relation": "targets"},
+            ],
+        },
+        {
+            "from": "intrusion-set",
+            "to": [
+                {"entity": "malware", "relation": "uses"},
+                {"entity": "vulnerability", "relation": "targets"},
+                {"entity": "attack-pattern", "relation": "uses"},
+                {"entity": "location", "relation": "targets"},
+                {"entity": "identity", "relation": "targets"},
+            ],
+        },
+        {
+            "from": "indicator",
+            "to": [
+                {"entity": "malware", "relation": "indicates"},
+                {"entity": "threat-actor", "relation": "indicates"},
```

Comment:
Did you miss `{"entity": "intrusion-set", "relation": "indicates"}` ? (based on you entities mapping relationships)

Reply:
Yes, thank you! 
Done :)